ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_sums_products_differences.py

```diff
@@ -61,3 +61,56 @@ def test_numpy_prod(
         where=where,
         test_values=True,
     )
+
+# sum
+@handle_cmd_line_args
+@given(
+    dtype_and_x=np_frontend_helpers.dtype_x_bounded_axis(
+        available_dtypes=ivy.current_backend().valid_dtypes
+    ),
+    dtype=st.sampled_from(ivy.current_backend().valid_dtypes + (None,)),
+    where=np_frontend_helpers.where(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.sum"
+    ),
+    keepdims=st.booleans(),
+    initial=st.one_of(st.booleans(), st.integers(), st.floats(), st.complex_numbers())
+)
+def test_numpy_sum(
+    dtype_and_x,
+    dtype,
+    where,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    fw,
+    keepdims,
+    initial
+):
+    input_dtype, x, axis = dtype_and_x
+    input_dtype = [input_dtype]
+    where = np_frontend_helpers.handle_where_and_array_bools(
+        where=where,
+        input_dtype=input_dtype,
+        as_variable=[as_variable],
+        native_array=[native_array],
+    )
+    np_frontend_helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="numpy",
+        fn_tree="sum",
+        x=np.asarray(x, dtype=input_dtype[0]),
+        axis=axis,
+        dtype=dtype,
+        out=None,
+        keepdims=keepdims,
+        initial=initial,
+        where=where,
+        test_values=True,
```

Comment:
please use `test_values=False` as values are tested differently (handled internally) with the presence of `where` argument